.gitignore

```diff
@@ -48,4 +48,9 @@ docs/source/tools/assets/
 .tfcache
 .env
 .python-version
-
+# files generated during "make test"
```

Comment:
Yeah these are from the kubernetes tests which need fixing, there's a separate open issue for that.

Reply:
Maybe keep this in the .gitignore for now? we can clean it up once the issue is fixed.

Reply:
sounds good